src/main/java/net/rpgtoolkit/common/utilities/TileSetCache.java

```diff
@@ -20,7 +27,7 @@
 public class TileSetCache {
 
   // Singleton.
-  private static final TileSetCache instance = new TileSetCache();
+  private static final TileSetCache INSTANCE = new TileSetCache();

```

Comment:
Converted it to a Static class a re-tested works the same as before. I think there is a slight performance boost when using static methods from what I can remember.